owrrm/src/main/java/com/facebook/openwifi/rrm/optimizers/channel/ChannelOptimizer.java

```diff
@@ -647,7 +728,7 @@ protected void logPerfMetrics(
 	 * @param configManager the ConfigManager instance
 	 * @param channelMap the map of devices (by serial number) to radio to channel
 	 */
-	public void updateDeviceApConfig(
+	public static void updateDeviceApConfig(
```

Comment:
Please keep this consistent. If you really want to change this, update all callers, and do the same for `TPC`. Otherwise leave it as is (there's not really any benefit to making it static).

Reply:
Its caller (same caller for TPC) cannot be made static, but I will update the TPC method as well.